plugins/services/src/js/containers/service-debug/ServiceDebugContainer.js

```diff
@@ -285,20 +285,27 @@ class ServiceDebugContainer extends React.Component {
     const waitingSince = DateUtil.strToMs(queue.since);
     const timeWaiting = Date.now() - waitingSince;
 
-    // If the service has been waiting for less than five minutes, we don't
-    // display the warning.
-    if (timeWaiting < 1000 * 60 * 5) {
-      return null;
-    }
-
-    let message, primaryAction;
+    let message,
+      primaryAction,
+      secondaryAction = null;
 
     if (service.isDelayed()) {
       message = (
         <Trans render="span">
           DC/OS has delayed the launching of this service due to failures.
         </Trans>
       );
+      secondaryAction = (
+        <Trans render="span">
+          <a
```

Comment:
It could, but it looks like a link in the designs: https://mesosphere.invisionapp.com/share/APRADUU3XKN#/screens/355292332 , so I decided to use a link for easier styling.

Reply:
If you think this is wrong, I will create a new PR to change it to a button.